### DISCUSSION TITLE
How do you use scrolls in VR?

### DISCUSSION THREAD

**Discussion by dlovinger** [Raicuparta/nomai-vr#508] [General]
I'm unable to use scrolls in VR. I try to insert a scroll to its slot on the wall, but nothing happens. I see no other ways of interacting, only B to drop the scroll. I have tried rotating the scroll in all directions, moving it in and out, etc. Is there something I'm missing?

**Comment by Raicuparta** [comment]
You always use the laser to interact, even while holding objects. So aim at the socket with the laser and press the interact button. You don't physically insert the scroll in the socket.

**Reply by isohpod** [reply]
And then how do you actually translate the writings?  I spent an entire loop in the Timber Hearth cave trying to translate the text on the scroll wall, every button combination imaginable, and got nowhere. 😭😭😭
